custom_components/easee/services.yaml

```diff
@@ -85,7 +85,7 @@ update_firmware:
       example: "EH123456"
 
 set_circuit_dynamic_current:
-  description: Set circuit dynamic current
+  description: Set circuit dynamic current (current available to share by several chargers on a circuit, to be set on master charger).
```

Comment:
The last part about the master charger is unclear to me. The API we call is not actually associated with the charger, so why would this be the case?

Reply:
It's according to documentation, seems like only the master charger is allowed to set dynamic circuit currents:

![image](https://user-images.githubusercontent.com/54450177/97792095-070c5000-1bda-11eb-8068-22cad631af6e.png)

Reply:
Hmm, you could be right for this one and not for the next comment. This specific one calls the circuitID so its not relevant, while the otherone is called with ChargerID.

Reply:
On third thought, you may be right since we actually just get the circuitID from the charger called. So we're in fact smarter than Easee here :) I'll remove it.

Reply:
Both of these services are exactly the same thing underneath.
The charger version is a wrapper for the circuit version.
The charger id is really totally unimportant for setting a circuit current.
The comment in the documentation does not mean that you need to know anything about how the structure of how your system is organised when you set it.
So my thinking is that it will confuse rather than help.

Reply:
Suggestion:
description: Set circuit dynamic current (current will be automatically shared by all chargers connected to the circuit)